examples/mouse_eye_movements.py

```diff
@@ -0,0 +1,300 @@
+fig, ax = plot_x_y_keypoints(
+    position_norm_dict, ylim=(-200, 200), time=time_points
+)
+fig.show()
+# %%
+# Pupil position over time
+# ------------------------
+# To look at pupil position, and later also velocity, over time we use the
+# pupil centroid (in this case the midpoint between keypoints ``pupil-L`` and
+# ``pupil-R``). The pupil centroid keypoint ``pupil-C`` is be added to the
+# data array using ``xarray.DataArray.assign_coords`` and ``xarray.concat``.
+for da_name, da in position_norm_dict.items():
+    pupil_centroid = da.sel(keypoints=["pupil-L", "pupil-R"]).mean("keypoints")
+    pupil_centroid = pupil_centroid.assign_coords({"keypoints": "pupil-C"})
+    position_norm_dict[da_name] = xr.concat([da, pupil_centroid], "keypoints")
+
+# %%
+# Now the position of the pupil centroid ``pupil-C`` can be plotted.
+fig, axes = plt.subplots(2, 1, figsize=(6, 4))
+for i, (da_name, da) in enumerate(position_norm_dict.items()):
+    da = da.sel(keypoints="pupil-C", time=time_points)  # select data to plot
+    for coord, color in ["x", "C0"], ["y", "C1"]:
+        axes[i].plot(
+            time_points,
+            da.sel(space=coord),
+            label=coord,
+            color=color,
+        )
+    axes[i].set_xlabel("Time (s)")
+    axes[i].set_ylabel("Position")
+    axes[i].set_title(f"Pupil centroid position ({da_name})")
+    axes[i].legend()
+
+for ax in axes.flat:
+    ax.set_ylim(-30, 65)
+
+plt.tight_layout()
+plt.show()
+# %%
+# Pupil velocity over time
+# ------------------------
+# We use ``compute_velocity`` from the ``movement.kinematics`` module to
+# calculate the velocity with which the pupil centroid moves.
+velocity_dict = {}
+for da_name, da in position_norm_dict.items():
+    velocity_dict[da_name] = kin.compute_velocity(da.sel(keypoints="pupil-C"))
+
+# %%
+# Now we can plot pupil velocity over time.
+fig, axes = plt.subplots(2, 1, figsize=(6, 4))
+for i, (da_name, da) in enumerate(velocity_dict.items()):
+    for coord, color in ["x", "C0"], ["y", "C1"]:
+        axes[i].plot(
+            time_points,
+            da.sel(space=coord, time=time_points),
+            label=coord,
+            color=color,
+        )
+    axes[i].set_xlabel("Time (s)")
+    axes[i].set_ylabel("Velocity")
+    axes[i].set_title(f"Pupil velocity ({da_name})")
+    axes[i].legend()
+
+plt.tight_layout()
+plt.show()
+# %%
+# The positive peaks correspond to rapid eye movements to the right, the
+# negative peaks correspond to rapid eye movements to the left.
+
+# %%
+# Pupil diameter
+# --------------
+# Here we define the pupil diameter as the distance between the two pupil
+# keypoints. We use ``compute_pairwise_distances`` from ``movement.kinematics``
+# to calculate the Euclidean distance between "pupil-L" and "pupil-R".
+pupil_diameter_dict = {}
+for da_name, da in position_norm_dict.items():
+    pupil_diameter_dict[da_name] = kin.compute_pairwise_distances(
+        da, "keypoints", {"pupil-L": "pupil-R"}
+    )
```

Comment:
Great job showcasing this function.
It's a very useful one and it hadn't yet made any appearance in our examples.
Isuggest adding the arg name to make it extra readable.

```suggestion
    pupil_diameter_dict[da_name] = kin.compute_pairwise_distances(
        da, dim="keypoints", pairs={"pupil-L": "pupil-R"}
    )
```